fili-core/src/test/resources/testApplicationConfig.properties

```diff
@@ -63,6 +63,3 @@ bard__default_per_page = 10000
 
 # Intersection reporting enabled or not.
 bard__intersection_reporting_enabled = false
-
-# Flag to turn on case sensitive keys in keyvalue store
-bard__case_sensitive_keys_enabled = false
```

Comment:
Why is this being removed here? Does this have anything to do with the rest of the changes in this PR?